### PR TITLE
23.10 / remove license requirement on k8s passthrough mode

### PR DESCRIPTION
**Pull request by Ornias1993**
While I 100% agree that "passthrough mode" should not exposed to normal end users directly in the GUI. Locking this behind enterprise hardware requirements, has no technical reasoning at all behind it. At the same time, it's already trivial to remove this needless requirement manually for people with the knowhow to even make the API call.

iX has always been clear that it only puts OS features and API calls behind a license if it's technically required.

**TLDR**
This limit isn't technically needed, it makes no sense for gatekeeping novice users AND it seems to be in contrast/violation of expected iX policies regarding licensing.

This is not likely to be used much at all, because the knowhow required to even correctly deploy this hidden gem, is huge.
Basically: lets keep it security through obscurity please.

### CODE REVIEW

**Comment by kmoore134** [comment]
We are not planning to remove this check. This is something we're only expecting to use for Enterprise Customers we are directly engaged with on their deployments.

**Comment by Ornias1993** [comment]
> We are not planning to remove this check. This is something we're only expecting to use for Enterprise Customers we are directly engaged with on their deployments.

So this means iX is officially droping the old policy of "we only license restrict where technical issues arise"?

**Comment by kmoore134** [comment]
I would argue that it fits this policy. It opens doors to a whole host of technical issues that we are not planning to support on the community side of the project, it would render the community edition unsupported.

**Comment by Ornias1993** [comment]
> I would argue that it fits this policy. It opens doors to a whole host of technical issues that we are not planning to support on the community side of the project, it would render the community edition unsupported.

And how is that related to hardware?
Like the fiber support and such?

THAT was the policy untill now.
Technical as in: Hardware related.

**Comment by Ornias1993** [comment]
From another side:
hostPath validation: "opens doors to a whole host of technical issues that we are not planning to support"(when disabled)
**Fully supported even in GUI**

This: "opens doors to a whole host of technical issues that we are not planning to support"
**Somehow super important to add license limitations for**
